### ISSUE TITLE
[FIGURE] A collapsing projection

### ISSUE THREAD

**Issue by SheepTester** [Gunn-GAtM/gatm#37]
**Which section?**
Composite Mappings of the Plane, problem 20

**Describe the improvement**
Someone on Facebook expressed confusion about the problem. I agree that the problem is a bit wordy. Perhaps there could be a small figure on the side that shows a selection of points moving on a perpendicular line to the collapsed line.

Also,
https://github.com/anematode/gatm/blob/7bd5a31fd3d7c253411ec51470436fdb8780dd44/comp_map/comp_map_source.tex#L603

> a point on a line perpendicular to the line has its image on the origin.

Do you mean a perpendicular line that goes through the origin? All points are on a line perpendicular to the line, but their image isn't necessarily on the origin.

**Comment by anematode** [comment]
Fixed in
